model/sharing/group.go

```diff
@@ -3,13 +3,15 @@ package sharing
 import (
 	"github.com/cozy/cozy-stack/model/contact"
 	"github.com/cozy/cozy-stack/model/instance"
+	"github.com/cozy/cozy-stack/pkg/couchdb"
 )
 
 // Group contains the information about a group of members of the sharing.
 type Group struct {
 	ID      string `json:"id,omitempty"` // Only present on the instance where the group was added
 	Name    string `json:"name"`
 	AddedBy int    `json:"addedBy"` // The index of the member who have added the group
+	Removed bool   `json:"removed,omitempty"`
```

Comment:
I feel like using the same name here would be better as it is the same concept, applied on different objects.
I don't think we could actually mistake one for the other since they're manipulated in different contexts.